dotcom-rendering/src/web/components/StickyVideo.importable.tsx

```diff
@@ -104,14 +108,38 @@ export const StickyVideo = ({ isPlaying, children }: Props) => {
 		setIsSticky(isPlaying && !isIntersecting);
 	}, [isIntersecting, isPlaying]);
 
+	useEffect(() => {
+		if (isSticky && wasClosed) {
+			setWasClosed(false);
```

Comment:
> just because I like the arrow function more

This is a perfectly fine and valid reason to use arrow functions! I guess I have the habit to use the older pattern for small, locally defined things but it's not for any good reason!